crates/pessimistic-proof/src/local_exit_tree/mod.rs

```diff
@@ -106,7 +113,7 @@ where
 
     /// Computes and returns the root of the tree.
     pub fn get_root(&self) -> H::Digest {
-        let mut root = H::Digest::default();
+        let mut root = H::Digest::default(); // The hash of the node we’re going to fill next.
```

Comment:
Nit: I generally prefer comments on a separate line, and so does a fair amount of tooling.